examples/nextjs/README.md

```diff
@@ -0,0 +1,38 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
What do you think about adding a branded header here together with a short note on running the app with docker compose as a prereq?